user_tools/src/spark_rapids_pytools/common/prop_manager.py

```diff
@@ -46,6 +46,14 @@ def to_camel_case(word: str) -> str:
     return res
 
 
+def get_gpu_device_list():
+    return ['T4', 'V100', 'K80', 'A100', 'P100', 'A10', 'A10G']
+
+
+def is_valid_gpu_device(val):
```

Comment:
specify return type to the method.
This makes the code easier to read and pylint will use those as hints to identify bugs in some cases.